mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/impl/DocumentIdProcessor.java

```diff
@@ -33,7 +34,14 @@ public void process(PropertyMappingStep mappingContext, DocumentId annotation,
 			PropertyMappingAnnotationProcessorContext context) {
 		IdentifierBinder binder = createIdentifierBinder( annotation, context );
 
-		mappingContext.documentId().identifierBinder( binder );
+		IdentifierBinderRef identifierBinderRef = annotation.identifierBinder();
+		if ( identifierBinderRef.params() != null ) {
+			Map<String, Object> params = ParameterAnnotationsReader.toMap( identifierBinderRef.params() );
```

Comment:
What's the point of exposing a method in the `context` if you don't use it? :)

```suggestion
			Map<String, Object> params = context.toMap( identifierBinderRef.params() );
```